src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/__init__.py

```diff
@@ -5,105 +5,186 @@
+            elif method_index == 3: # skip
+                return
+            try:
+                profile.find_subscriptions_on_login(
+                    interactive,
+                    username,
+                    password,
+                    service_principal,
+                    tenant)
+                login_successful = True
+                logger.warning('Login successful!')
+            except AdalError as err:
+                logger.error('Login error!')
+                logger.error(err)
+
+def _create_or_update_env(env_name=None):
+    if not env_name:
+        # TODO Support new env creation
+        print('This feature is coming soon.\n')
+        sys.exit()
+    # get the config parser for this env
+    env_config = configparser.SafeConfigParser()
+    env_config.read(os.path.join(ENV_CONFIG_DIR, env_name))
+    # prompt user to choose cloud for env
+    selected_cloud_index = prompt_choice_list(MSG_PROMPT_WHICH_CLOUD, CLOUD_LIST,
+                                              default=get_default_from_config(env_config,
+                                                                              'cloud',
+                                                                              'name',
+                                                                              CLOUD_LIST))
+    if CLOUD_LIST[selected_cloud_index]['name'] != 'public-azure':
+        # TODO support other clouds
+        print('Support for other clouds is coming soon.\n')
+        sys.exit()
+    try:
+        env_config.add_section('cloud')
+    except configparser.DuplicateSectionError:
+        pass
+    env_config.set('cloud', 'name', CLOUD_LIST[selected_cloud_index]['name'])
+    # TODO when we support other clouds, extend this to a class. Keeping it simple for now.
+    _config_env_public_azure(env_config)
+    # save the config
+    if not os.path.isdir(ENV_CONFIG_DIR):
+        os.makedirs(ENV_CONFIG_DIR)
+    with open(os.path.join(ENV_CONFIG_DIR, env_name), 'w') as configfile:
+        env_config.write(configfile)
 
 def _handle_global_configuration():
-    print(MESSAGES_GLOBAL_SETTINGS_LOCATION.format(GLOBAL_CONFIG_PATH))
-    print(MESSAGES_ACTIVE_ENV_SETTINGS_LOCATION.format(ACTIVE_ENV_CONFIG_PATH))
-    # We don't want to use AzConfig as we don't want to use values set by env vars.
+    # print location of global configuration
+    print(MSG_GLOBAL_SETTINGS_LOCATION.format(GLOBAL_CONFIG_PATH))
+    if os.path.isfile(ACTIVE_ENV_CONFIG_PATH):
+        # print location of the active env configuration if it exists
+        print(MSG_ACTIVE_ENV_SETTINGS_LOCATION.format(ACTIVE_ENV_CONFIG_PATH))
+    # set up the config parsers
     file_config = configparser.SafeConfigParser()
     config_exists = file_config.read([GLOBAL_CONFIG_PATH, ACTIVE_ENV_CONFIG_PATH])
     global_config = configparser.SafeConfigParser()
     global_config.read(GLOBAL_CONFIG_PATH)
-    change_global_settings = None
+    should_modify_global_config = False
```

Comment:
I might not got it right still, but should this  be True? So at least when no config exists, we will save it to the global one.